### ISSUE TITLE
Randomized Skyboxes

### ISSUE THREAD

**Issue by cswimr** [GalaxyRoblox/Issues-Public#71]
This issue will have two parts:

## Randomized Skyboxes

Whenever a server is started, select a skybox from a list of default skyboxes. This skybox will be used as the "Default"/"Random" skybox for the duration of the server, if Player-Selectable Skyboxes were implemented as well. This would provide a feeling of variety between servers, as they look visually distinct from eachother.

## ~~Player-Selectable Skyboxes~~

~~Add an option to the settings menu that allows a player to select a skybox from a list of skyboxes, instead of just using the normal skybox. This allows players to customize their game to look how they want it to look. If there are no moderation concerns, we could also allow players to use other skyboxes from the asset library.~~

## Discord Suggestion Numbers
- [2227](https://discord.com/channels/204965774618656769/1047550652873711697/1263957149600452608) - `legs_v  (801381324949815306)`
- [2228](https://discord.com/channels/204965774618656769/1047550652873711697/1263957189983342613) - `seaswimmer  (294518358420750336)`

## Edit

Player-Selectable Skyboxes have been scrapped. If we still want these, we can make a separate issue for them later on.

**Comment by Wingysam** [comment]
We could also use skyboxes seasonally. For example, we could use a red/green nebula during Christmas or similar. Keeping skyboxes dev-controlled allows us some creative expression.

**Comment by cswimr** [comment]
Good point, maybe there could be an option within EventID configurations to override the randomized skybox? If people wanted to choose a skybox in the main game, they still could, but the default would be whatever the EventID configuration specifies.
